pytest/lib/cluster.py

```diff
@@ -900,6 +905,20 @@ def init_cluster(
 
     logger.info("Search for stdout and stderr in %s" % node_dirs)
 
+    # if extra_state_dumper is True, we added 1 to num_observers above and we will enable
+    # state dumping to a local tmp dir on the last node in node_dirs. The other nodes will have their
+    # state_sync configs point to this tmp dir
+    # TODO: remove this extra_state_dumper option when centralized state sync is no longer used
+    if extra_state_dumper:
+        (node_config_dump,
+         node_config_sync) = state_sync_lib.get_state_sync_configs_pair(
```

Comment:
as follow up, I think we can start using `extra_state_dumper` for all the tests that use `get_state_sync_config_pairs` now.